### PR TITLE
Use Icarus and migrate to npm

### PR DESCRIPTION
**Pull request by pawalt**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,12 +6,12 @@ WORKDIR /app/
 
 # Copy project dependencies
 COPY package*.json /app/
-COPY yarn.lock /app/
+COPY package-lock.json /app/
```
Comment (ArmaanT) [inline Dockerfile]: This line is redundant with the one above, right?

File: k8s/values.yml [hunk 2]
```diff
@@ -1,16 +1,14 @@
...
-  domain: example.com
-
-secrets:
-  secretEnv: example
+deploy_version: 0.1.1
+image_tag: latest
+
+applications:
+  - name: worker
+    image: pennlabs/<image_name>
```
Comment (ArmaanT) [inline k8s/values.yml]: Replace `<image_name>` with `image-name` so the init script can update the values automaticaly.

**Review by ArmaanT — CHANGES_REQUESTED** [review]
Most of the changes look good. Has this been tested using basics or something?

File: k8s/values.yml (hunk 2 above)
Comment (pawalt) [reply k8s/values.yml]: ah thanks. wrong dash

File: Dockerfile (hunk 1 above)
Comment (pawalt) [reply Dockerfile]: good catch

**Comment by pawalt** [comment]
Yep basics is using this exact configuration right now.

**Review by ArmaanT — APPROVED** [review]
LGTM!

**Pull request merged**